graphql_compiler/compiler/subclass.py

```diff
@@ -37,7 +27,8 @@ def compute_subclass_sets(schema, type_equivalence_hints=None):
     # A class is a subclass of itself
     subclass_set = {
         classname: {classname}
-        for classname in six.iterkeys(schema.get_type_map())
+        for classname, graphql_type in six.iterkeys(schema.get_type_map())
+        if isinstance(graphql_type, (GraphQLInterfaceType, GraphQLObjectType))
```

Comment:
According to the docstring above:
"A class here means an object type or interface."